### ISSUE TITLE
window is not defined error

### ISSUE THREAD

**Issue by Tylertron1998** [walmik/scribbletune#65]
Hi there; recently picked this up as I wanted to play about with some audio generation - so.

I assume from the error I am getting, this is made to run in the browser? As running it via node gives me the above error. Is there anything I can do for this?

Thank you in advance.

**Comment by Yukippe** [comment]
I also encountered this error
```
.../node_modules/tone/build/Tone.js:633
	            var hasAudioContext = window.hasOwnProperty('AudioContext') || window.hasOwnProperty('webkitAudioContext');
	                                  ^

ReferenceError: window is not defined
```
hope to get your help.  thanks

**Comment by Yukippe** [comment]
@Tylertron1998 .
 had try version 0.11.2. it's running well

**Comment by walmik** [comment]
Crudely fixed in `1.0.0-alpha.2`
